packages/launcher/style/base.css

```diff
@@ -163,8 +163,8 @@
   word-break: break-word;
   text-align: center;
   color: var(--jp-ui-font-color1);
-  line-height: 14px;
-  font-size: 12px;
+  line-height: var(--jp-ui-font-size1); /*initially was 14px*/
```

Comment:
I would change the `line-height` to be proportional with the `font-size` to keep the default ratio. See https://developer.mozilla.org/en-US/docs/Web/CSS/line-height